packages/cli/generators/app/templates/src/application.ts.ejs

```diff
@@ -19,6 +19,21 @@ export class <%= project.applicationName %> extends BootMixin(
 export class <%= project.applicationName %> extends BootMixin(RestApplication) {
 <% } -%>
   constructor(options: ApplicationConfig = {}) {
+    options = Object.assign(
+      {
+        rest: {
+          // your application listening port number, you can change this once
+          // in production
+          port: 3000,
+          openApiSpec: {
+            // useful when used with OASGraph to locate your application when
+            // resolving GraphQL schemas
+            servers: [{url: 'http://localhost:3000'}],
```

Comment:
Is `setServersFromRequest: true` better as we don't always know the port in advance?

Reply:
If you have `setServersFromRequest: true`, what's `servers` from the generated openapi spec?

Please note that browsers may cache the response. I had to use private browsing sometimes to work around it.

Reply:
> If you have `setServersFromRequest: true`, what's `servers` from the generated openapi spec?

```
"servers": [
    {
      "url": "http://[::1]:3010"
    }
  ]
```
However, the `node_modules/oasgraph/cli.js http://localhost:3010/openapi.json` still doesn't work. I guess the problem is with OASGraph not liking the format [::1]  ?

Reply:
Neither `node_modules/.bin/oasgraph http://localhost:3000/openapi.json`  or probably the problem is the `express-graphql` since this is the one connecting the resolver from the GraphQL schema.

Reply:
`http://[::1]:3010` uses ipv6 address.